VSCode_format_guide.md

```diff
@@ -0,0 +1,20 @@
+# How to Format to Google Java Style using VSCODE:
+1. Install VSCode
```

Comment:
Maybe we could link [VSCodium](https://github.com/VSCodium/vscodium) instead? It is the libre version of VSCode and since we are an open source project I really would like to stick to open source software!